samples/word/04-range/split-words-of-first-paragraph.yaml

```diff
@@ -4,157 +4,84 @@ description: 'Show how to split a paragraph into word ranges and then traverses
-                var myPars = context.document.body.paragraphs;
-                context.load(myPars);
-                return context.sync()
-                    .then(function () {
-                        var myWords = myPars.items[0].split([" "], true /*used to trim delimiters*/, true /* used to trim spaces */);
-                        context.load(myWords, { expand: 'font' });
-                        return context.sync()
-                            .then(function () {
-                                return forEach(myWords, function (item, i) {
-                                    if (i >= 1) {
-                                        myWords.items[i - 1].font.highlightColor = "#FFFFFF";
-                                    }
-                                    myWords.items[i].font.highlightColor = "#FFFF00";
-                                    return createTimerPromise(200).then(context.sync);
-                                })
-                            })
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
-
-            function createTimerPromise(ms) {
-                return new OfficeExtension.Promise(function (resolve) {
-                    setTimeout(resolve, ms);
-                })
-            }
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#highlight").click(() => tryCatch(highlightWords));
+
+        async function highlightWords() {
+            await Word.run(async (context) => {
+                let paragraph = context.document.body.paragraphs.getFirst();
+                let words = paragraph.split(
+                    [" "], true /* trimDelimiters*/, true /* trimSpaces */);
+                words.load("text");
+                await context.sync();
+
+                for (let i = 0; i < words.items.length; i++) {
+                    if (i >= 1) {
+                        words.items[i - 1].font.highlightColor = "#FFFFFF";
+                    }
+                    words.items[i].font.highlightColor = "#FFFF00";
+                    await context.sync();
+                    await pause(200);
+                }
+            });
+        }
 
-            function forEach(collection, handler) {
-                var promise = new OfficeExtension.Promise(function (resolve) { resolve(); });
-                collection.items.forEach(function (item, index) {
-                    promise = promise.then(function () {
-                        return handler(item, index);
-                    })
-                });
-                return promise;
-            }
+        function pause(milliseconds) {
+            return new Promise(resolve => setTimeout(resolve, milliseconds));
         }
 
-        function setup() {
-            Word.run(function (context) {
+        async function setup() {
+            await Word.run(async (context) => {
                 // lets insert a couple of paragraphs to illustrate the point..
```

Comment:
I realize you didn't write this comment, but can you edit it? :)   

- remove **lets**
- capitalize **Insert**
- remove the duplicate period from the end of the sentence